.github/ISSUE_TEMPLATE/onboarding-checklist.md

```diff
@@ -52,20 +47,40 @@ Feel free to ask anyone in the group questions about these steps, especially in
     -   [ ] Select and enroll in a retirement plan. You need to enroll in retirment within 30 days of your start date.
     -   [ ] Once you have selected your retirement plan, you can sign up for other benefits. Enroll in other benefits within 31 days of starting.
 
-### Internet
+### Equipment setup
+
+#### Internet
 
 You should be on the university's VPN any time you connect to internet outside the UA.
 
 -   [ ] Install Cisco Secure Client; instructions under [Virtual Private Network (VPN)](https://it.arizona.edu/remote-work).
     -   When asked for a third password/NetID+ method, input "push" or "sms" or "passcode" to autheticate via 2FA.
 -   [ ] Log in to the VPN by opening Cisco Secure Client, typing in "vpn.arizona.edu/cals-ext", and clicking Connect. Then log in using your UA credentials and 2FA.
 
+#### Software set up
+
+When you get your computer...
+
+-   [ ] Meet with an admin in CCT (Martin Picazzo as of 2023) to get logged into your computer the first time.
+-   [ ] Also install software you will need administrative privileges to install. It can be helpful to come to this meeting with a list of things you expect to use. In future, if you need an admin password to install something, contact Martin to get a temporary password.
```

Comment:
```suggestion
-   [ ] Also install software that you will need administrative privileges to install. It can be helpful to come to this meeting with the CCT admin having a list of things you expect to use. In the future, if you need an admin password to install something, contact Martin to get a temporary password.
```